specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2022-01-01-preview/namespace-preview.json

```diff
@@ -439,6 +439,12 @@
               "$ref": "#/definitions/PrivateEndpointConnection"
             }
           },
+          "202": {
+            "description": "Request to update Status of PrivateEndPoint Connection accepted.",
+            "schema": {
+              "$ref": "#/definitions/PrivateEndpointConnection"
+            }
+          },
```

Comment:
PUTs should not be returning 202s. Why do you need to make this change in existing API ?  : 

Please read through the details here : 
[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creating-or-updating-resources-asynchronously](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creating-or-updating-resources-asynchronously)

Reply:
@raosuhas , we have made this change in a previous api version as well in this PR https://github.com/Azure/azure-rest-api-specs/pull/19155 and no questions were raised. The reason we made this change is that otherwise it renders out CreateOrUpdate command useless. We have already released the fix in 2021-11-01 SDK, please allow it in 2022-01-01-preview as well.